src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -721,9 +721,23 @@
      * @see FullIdent
      **/
     public static final int METHOD_CALL = GeneratedJavaTokenTypes.METHOD_CALL;
-
     /**
      * Part of Java 8 syntax. Method or constructor call without arguments.
```

Comment:
`call` is incorrect, right? It should be `reference`. We are not actually calling the methods with this, just referencing it.